docs/build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM projectatomic/ascii_binder
```

Comment:
This wasn't originally licensed by Kubernetes, this is something new. Do we still need the previous license?